app.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
```

Comment:
- [ ] When I run the application, It throws an error. There's a typo, Please change `require_relative 'student'` to `require_relative 'students'`.